sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -307,6 +307,7 @@ public partial class AzureTableDataFeedSource : Azure.AI.MetricsAdvisor.Administ
     public abstract partial class DataFeedSource
     {
         internal DataFeedSource() { }
+        public Azure.AI.MetricsAdvisor.Models.DataFeedSourceType DataSourceType { get { throw null; } }
```

Comment:
Naming it `DataSourceType` for consistency with other languages. Considered proposing a rename to `Type`, but not worth it in our current time frame.